props/saber_BC_buttons.h

```diff
@@ -538,24 +538,30 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
   void QuickMaxVolume() {
     SaberBase::DoEffect(EFFECT_VOLUME_LEVEL, 0);
     dynamic_mixer.set_volume(VOLUME);
-    if (SFX_volmax) {
-      hybrid_font.PlayPolyphonic(&SFX_volmax);
-    } else {
-      beeper.Beep(0.5, 3000);
+    if (millis() - volume_range_delay_ > 2000) {
+      if (SFX_volmax) {
+        hybrid_font.PlayPolyphonic(&SFX_volmax);
+      } else {
+        beeper.Beep(0.5, 3000);
+      }
+      PVLOG_NORMAL << "Maximum Volume\n";
+      volume_range_delay_ = millis();
     }
-    PVLOG_NORMAL << "Maximum Volume\n";
   }
 
   void QuickMinVolume() {
     SaberBase::DoEffect(EFFECT_VOLUME_LEVEL, 0);
     dynamic_mixer.set_volume(std::max<int>(VOLUME * 0.1,
     dynamic_mixer.get_volume() - VOLUME * 0.90));
-    if (SFX_volmin) {
-      hybrid_font.PlayPolyphonic(&SFX_volmin);
-      } else {
-        beeper.Beep(0.5, 1000);
-      }
+    if (millis() - volume_range_delay_ > 2000) {
+      if (SFX_volmin) {
+        hybrid_font.PlayPolyphonic(&SFX_volmin);
+        } else {
```

Comment:
This line (and the next 2) look weirdly indented.
Not sure if  if it's just github being weird or not.

Reply:
Weird, it's not like that on my end. 
Anyway, fixed, thanks.